docs/intellicode-visual-studio-code.md

```diff
@@ -16,24 +16,43 @@ The [IntelliCode extension for Visual Studio Code](https://marketplace.visualstu
 
 ![IntelliCode for Python in Visual Studio Code](media/python-intellicode.gif)
 
-## Supported languages
-
 The supported languages for AI-assisted IntelliSense completions are:
 
 - Java
 - JavaScript and TypeScript
 - Python
 - SQL
 
-## Prerequisites
 
-To install the extension, you need Visual Studio Code October 2018 Release 1.29 or later.
+### Prerequisites
+
+The extension is supported by Visual Studio Code October 2018 Release 1.29 or later.
```

Comment:
I don't think this is true, I think we require version 1.58 or later for the main IntelliCode extension, and we also require 1.63 or later for the Completions extension (this may be bumped to 1.67 or so once we support brace completion)

Reply:
/cc @juihanamshet1

Reply:
We're still supporting 1.63 with the old API. The new API is for 1.67 onwards. But VS Code stable hasn't released 1.67 yet, still in Insiders

Reply:
Good thing we're going through these old docs :)